authors/apps/articles/models.py

```diff
@@ -37,3 +39,16 @@ def slug_generator(sender, instance, *args, **kwargs):
 
 
 pre_save.connect(slug_generator, sender=Article)
+
+
+class Likes(models.Model):
```

Comment:
@KabohaJeanMark You should provide a more descriptive name since this model caters for dislikes and likes. 
```suggestion
class Likes_Dislikes(models.Model):
```

Reply:
Yes. I have edited this